MockGUI/GolemModel.cs

```diff
@@ -201,12 +202,14 @@ public async void OnListJobs()
             List<IJob> jobs;
             try
             {
+                _logger.LogInformation("Listing jobs since: " + since);
                 jobs = await this.Golem.ListJobs(since);
-            } catch (Exception)
+            }
+            catch (Exception)
             {
```

Comment:
Swallowing it will lead to difficult to debug _empty jobs list_ issues.
```suggestion
            catch (Exception)
            {
                _logger.LogWarning(e, "Listing jobs failure");
```